brjs-core/src/main/java/org/bladerunnerjs/model/AppVersionGenerator.java

```diff
@@ -3,7 +3,9 @@
 
 public interface AppVersionGenerator
```

Comment:
It's not clear to me why we need `dev` and `prod` versions? Isn't it the case that the version string just tends to have a different form when run in `dev` and `prod`?